charts/ethereum-node/values.yaml

```diff
@@ -93,6 +93,7 @@ besu:
   enabled: false
   nameOverride: execution
   httpPort: 8545
+  p2pPort: 30303
```

Comment:
Any reason this or even the httpPort value is needed here?

Reply:
It's just to enforce it.